### PR TITLE
Move handlers to withSaga

### PR DESCRIPTION
**Pull request by ditkachuk**

### CODE REVIEW

File: src/withAsyncHandlers.js [hunk 1]
```diff
@@ -1,24 +1,46 @@
...
-            forEachObjIndexed((handlers, key) => {
-                if (this.props[key] && this.props[key].meta) {
-                    const { success : successHandler, error : errorHandler, pending : pendingHandler } = handlers;
-                    const { success, error, pending } = this.props[key].meta;
-                    const { meta } = nextProps[key];
+        constructor(props) {
+            super(props);
+            const sagas = [];
+
+            forEachObjIndexed(when(prop('saga'), (action) => {
```
Comment (megazoll) [inline src/withAsyncHandlers.js]: seems like you no need index here

Comment (megazoll) [reply src/withAsyncHandlers.js]: const sagas = map(...)

**Comment by megazoll** [comment]
bump version here

File: src/withAsyncHandlers.js (hunk 1 above)
Comment (ditkachuk) [reply src/withAsyncHandlers.js]: moved to next `forEachObjIndexed` for only one actions handler

**Comment by ditkachuk** [comment]
Some readme added

**Pull request merged**